src/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -421,3 +421,9 @@ document.body.addEventListener(
 ```
 
 Remember to pass in `true` as the second parameter to `addEventListener()`. Without it, the event handler won't be called, since it's being added to the event target's ancestor rather than the event target itself, and unlike JavaScript runtime errors, `error` events resulting from load failures don't bubble, and therefore must be captured during the `capture` phase. For more information, see [the W3C spec](https://www.w3.org/TR/2003/NOTE-DOM-Level-3-Events-20031107/events.html#Events-phases).
+
+## Supporting Older Browsers
+
+Starting with version 7.0.0, the Sentry JavaScript SDK uses ES6 syntax, along with a few other ES6+ language features like object spread. If you are down-compiling your code in order to target older browsers that don't support such syntax, you'll need to include the Sentry SDK in that process.
```

Comment:
We can also leave a note here to use an ES5 CDN bundle, since our default is now ES6

Reply:
Added a note.

Reply:
Wondering if it would work to make this a platform section that's only supported for next.js and then you don't need the include that's for all the other JS frameworks (if the only reason it's been provided is so this page displays properly for the rest of JS). Not a hard requirement for this PR, but might be worth checking.

Reply:
Yeah, I wasn't super clear in my wording, sorry about that. Everyone but nextjs should have thing A (which actually _is_ meaningful), nextjs should have thing B. I'll fix the PR description.